src/engine/plugins/trip.cpp

```diff
@@ -187,8 +204,90 @@ Status TripPlugin::HandleRequest(const std::shared_ptr<const datafacade::BaseDat
     BOOST_ASSERT_MSG(result_table.size() == number_of_locations * number_of_locations,
                      "Distance Table has wrong size");
 
+    // The following code manipulates the table and produces the new table for
+    // Trip with Fixed Start and End (TFSE)
+
+    // Original Table
+    //   a  b  c  d  e
+    // a 0  15 36 34 30
+    // b 15 0  25 30 34
+    // c 36 25 0  18 32
+    // d 34 30 18 0  15
+    // e 30 34 32 15 0
+
+    // New Table
+    //   a        b         c        d         e
+    // a 0 0      1 15      2 10000  3 34      4 30
+    // b 5 10000  6 0       7 25     8 30      9 34
+    // c 10 0     11 10000  12 0     13 10000  14 10000
+    // d 15 10000 16 30     17 18    18 0      19 15
+    // e 20 10000 21 34     22 32    23 15     24 0
+
+    std::vector<EdgeWeight> tfse_table_ = result_table.GetTable();
+
+    if (parameters.source > -1 && parameters.destination > -1)
+    {
+        for (std::size_t f_counter = 0; f_counter < tfse_table_.size(); f_counter++)
+        {
+            // parameters.source column
+            if (f_counter % result_table.GetNumberOfNodes() == (unsigned long)parameters.source)
+            {
+                tfse_table_[f_counter] = std::numeric_limits<EdgeWeight>::max();
```

Comment:
Given that the FI and BruteForce searches are looking for `INVALID_EDGE_WEIGHT`, that's what you should set here, not `::max()`.  Practically, they're the same thing, but if the `INvALID_EDGE_WEIGHT` value changes for some reason, the `if()` logic later on will break.